Gemfile

```diff
@@ -89,6 +86,7 @@ end
 group :production do
   # Add bootsnap for 5.2
   gem 'bootsnap', '~> 1.3'
+  gem 'unicorn'
```

Comment:
Whilst your at it, add a version spec to this to lock it down ?

Reply:
Fixed in 5b51a0a